analytics/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+
+from zerver.models import Realm, UserProfile
+from zerver.lib.str_utils import ModelReprMixin
+
+# only would ever touch by hand
+class Anomaly(ModelReprMixin, models.Model):
+    info = models.CharField(max_length=1000) # type: text_type
+
+    def __unicode__(self):
+        # type: () -> text_type
+        return u"<Anamoly: %s... %s>" % (self.info, self.id)
+
+class RealmCount(ModelReprMixin, models.Model):
+    realm = models.ForeignKey(Realm) # type: Realm
+    value = models.BigIntegerField() # type: int
+    property = models.CharField(max_length=40) # type: text_type
+    end_time = models.DateTimeField() # type: datetime.datetime
+    interval = models.CharField(max_length=20) # type: text_type
```

Comment:
if one wanted to optimize this, both `property` and `interval` could be enums.  See for example the `class Recipient` definition.